addons/website/static/src/client_actions/website_preview/website_preview.js

```diff
@@ -102,7 +102,7 @@ export class WebsitePreview extends Component {
                 this.websiteService.showLoader({ showTips: true });
             }
         }, () => [this.props.action.context.params]);
-        
+
```

Comment:
That's a random change 😆 Was this on purpose?

Reply:
Yes of course :eyes: